carmajava/signal_plugin/src/main/java/gov/dot/fhwa/saxton/carma/signal_plugin/ead/PlanInterpolator.java

```diff
@@ -68,23 +69,33 @@
       double v_old = v_0;
       double v = Math.sqrt(v_old * v_old + two_a_x);
       double t;
+      double x = x_0 + distanceStep;
```

Comment:
In line 63, after we added the node directly, should we just go to the next node?